AirCasting/SessionViews/SessionCartViewModel/DefaultDeleteSessionViewModel.swift

```diff
@@ -4,46 +4,106 @@
 import Foundation
 
 class DefaultDeleteSessionViewModel: DeleteSessionViewModel {
+    private let measurementStreamStorage: MeasurementStreamStorage
+    private var session: SessionEntity
+    private let streamRemover: StreamRemover
+    @Published var showingAlert: Bool = false
```

Comment:
please change name, I have no idea what alert you're referring to by this name (showingSomethingAlert)